src/code/code_n64dd_800AD410.c

```diff
@@ -12,6 +12,7 @@ s8 func_801C7740_unknown(void);
 u8 B_80121AE0;
 s8 B_80121AE1;
 u8 B_80121AE2;
+vu8 B_80121AE4;
```

Comment:
shouldn't there be a (unused so far) `B_80121AE3` for this added bss variable to be xxx4

Reply:
Good point. Apparently it's used in the n64dd segment as a `vu8`
https://github.com/Decompollaborate/n64dd/blob/6e138715c991bffb26216370c7fa8ebefbb065f9/oot/ne2/undefined_syms_ne2.txt#L55
https://github.com/Decompollaborate/n64dd/blob/6e138715c991bffb26216370c7fa8ebefbb065f9/include/code_functions.h#L33